webapp/src/pages/Users/Users.tsx

```diff
@@ -0,0 +1,14 @@
+import { IconVariant, NavigationItem } from '../../shared/components';
+import './Users.css';
+
+export default function Users() {
+  return (
+    <div className="users">
+      <NavigationItem
+        iconVariant={IconVariant.USERS}
+        title="Users"
+        urlNavigation="/users"
+      />
+    </div>
+  );
+}
```

Comment:
This page will be the one showing all the user in the future, won't it? And the NavItem would be part of the header.

Reply:
Yes, you are right.